org.osgi.test.junit4/src/main/java/org/osgi/test/common/annotation/Property.java

```diff
@@ -91,6 +91,21 @@ public enum ValueSource {
 		TestUniqueId
 	}
 
+	/**
+	 * Used to provide an argument in the {@link Property#templateArguments()}.
+	 * {@link ValueArgument} values are resolved in the same way as
+	 * {@link Property} values but they are always of {@link Type#Scalar}
+	 *
+	 * @since 1.2
+	 */
+	public @interface ValueArgument {
+		String[] value() default "";
+
+		Scalar scalar() default Scalar.String;
```

Comment:
This does not seem to be used or tested.

Reply:
I thought I had used it here:

https://github.com/osgi/osgi-test/pull/814/files#diff-14565fbe3b31cfc388c58178e1b351bc2a293b8eeaaf193a0af057125cfa9c42R363-R370

I'll fix up the test.